api/apiclient_test.go

```diff
@@ -1125,6 +1240,10 @@ type loginTimeoutAPIAdmin struct {
 }
 
 func (a *loginTimeoutAPIAdmin) Login(req params.LoginRequest) (params.LoginResult, error) {
-	<-a.r.unblock
+	ch, ok := <-a.r.unblock
```

Comment:
This is 2 cases of bare channel sends without an associated select + a tomb so that we can close the object.
Is this sane to be doing?
(I realize the original code was doing a bare receive as well.)
But its often a big red-flag for potential deadlocks if there are any bugs in other code.